common/src/main/java/com/github/khanshoaib3/minecraft_access/features/HealthAndHunger.java

```diff
@@ -0,0 +1,31 @@
+package com.github.khanshoaib3.minecraft_access.features;
+
+import com.github.khanshoaib3.minecraft_access.MainClass;
+import com.github.khanshoaib3.minecraft_access.utils.KeyBindingsHandler;
+import com.github.khanshoaib3.minecraft_access.utils.PlayerUtils;
+import com.github.khanshoaib3.minecraft_access.utils.condition.Keystroke;
+import lombok.extern.slf4j.Slf4j;
+import net.minecraft.client.resource.language.I18n;
+
+/**
+ * Speak Player Health and Hunger Key (default: R)
+ */
+@Slf4j
+public class HealthAndHunger {
+    private static final Keystroke key = new Keystroke(KeyBindingsHandler.getInstance().healthNHungerNarrationKey);
```

Comment:
Should the key be renamed to be consistent with everything else?
```suggestion
    private static final Keystroke key = new Keystroke(KeyBindingsHandler.getInstance().healthAndHungerNarrationKey);
```